src/routes/safe/components/Transactions/TxList/TxExpandedActions.tsx

```diff
@@ -41,6 +41,11 @@ export const TxExpandedActions = ({ transaction }: TxExpandedActionsProps): Reac
   // https://github.com/facebook/react/issues/4492
   return (
     <>
+      {canCancel && (
```

Comment:
Did you also change the collapsed actions to reflect this reordering?

Reply:
I will. Excellent point!